deploy/scripts/src/cri-lime/test.ts

```diff
@@ -9,6 +9,7 @@ import exec from 'k6/execution'
 import { timeRequest } from '../common/utils/request/timing'
 import { isStatusCode200, isStatusCode302, pageContentCheck } from '../common/utils/checks/assertions'
 import { sleepBetween } from '../common/utils/sleep/sleepBetween'
+import { getEnv } from '../common/utils/config/environment-variables'
```

Comment:
I have included this change in `[PR-544](https://github.com/govuk-one-login/performance-testing/pull/544)`. We can keep the scope of this PR to utility function documentation and merge this change via the other PR if that's okay.

Reply:
These changes are actually needed to make sure the TsDoc builds, it shouldn't matter which PR it is merged in. Might resolve in a merge conflict that needs resolution, but it should be simple to resolve